src/main/java/mate/academy/springboot/swagger/service/SortService.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.springboot.swagger.service;
+
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public interface SortService {
+    List<Sort.Order> sort(String sortBy);

```

Comment:
```suggestion
    List<Sort.Order> getSortOrders(String sortBy);

```
It will be more infomative than just sort(it means you are going to sort something). And, probably, you should move this method to util class not, in service.